Src/DependencyCollector/Shared/Implementation/ProfilerSqlProcessingBase.cs

```diff
@@ -371,7 +381,8 @@ private void OnEnd(object exceptionObj, object thisObj)
         /// </summary>
         /// <param name="exceptionObj">The exception object if any.</param>
         /// <param name="thisObj">This object.</param>
-        private void OnEndInternal(object exceptionObj, object thisObj)
+        /// <param name="processTelemetryItem">True if telemetry item should be processed.</param>
```

Comment:
name is not clear. Can you call it `sendTelemetryItem` instead. Description for `param` should also be more specific